### DISCUSSION TITLE
What kind of contributions are you looking for? How can I help?

### DISCUSSION THREAD

**Discussion by carlosaherrera** [vlang/v#21742] [General]
Coming from a Python and Javascript background, V has caught my attention. I find its design makes coding a delight while adding more robustness to the integrity of the code. My favorite thing about the language is Option/Result types and **forced** error handling. 

With that said, I'd like to contribute. Maybe there is some way I can dip my toes into the world of programming language software development. I come from a web development background. I am part way through completing craftinginterpreters.com. So my understanding of how languages are created is sparse but not entirely void. 

Need help with documentation? Something else perhaps?

**Comment by JalonSolov** [comment]
Everything is useful in an open source project.

Anything from simply correcting a typo, to rewriting documentation bits to make them more clear/concise... up to doing performance testing and speeding up library routines, contributing to the JavaScript backend, working on a Python -> V translator, or whatever else interests you.

As we say around here... PRs are always welcome.  :-)

As for how to make a contribution, check https://github.com/vlang/v/blob/master/CONTRIBUTING.md, come discuss things on our Discord server or in Discussions here, etc.
